python/examples/rviz_visual_tools_demo.py

```diff
@@ -0,0 +1,144 @@
+#!/usr/bin/env python3
```

Comment:
Can we add a note here and in the README that the python demo shows a smaller example of the displayable shapes (compared to the cpp version) in Rviz so users don't think it's missing stuff.